### PR TITLE
Sb 02

### PR DESCRIPTION
**Pull request by Rudiachenko**

### CODE REVIEW

File: src/main/java/Main.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        CsvFileReaderImpl csvFileReader = new CsvFileReaderImpl();
+        DataMapper dataMapper = new DataMapper();
+        DataParserImpl dataParser = new DataParserImpl(csvFileReader, dataMapper);
+        List<Comment> comments = dataParser.parseData();
+        for (Comment comment : comments) {
+            System.out.println(comment);
+            System.out.println();
+        }
+    }
+}
```
Comment (DmGrt) [inline src/main/java/Main.java]: I think we don't need this class here

**Review by DmGrt — COMMENTED** [review]
Good job but smth wrong with Travis

File: src/main/java/Main.java (hunk 1 above)
Comment (Rudiachenko) [reply src/main/java/Main.java]: Oops. This was for a test :D

**Review by KseniiaMakarova — COMMENTED** [review]
the first PR is not merged, so this PR contains some classes from the first one

also, I can see that the `reviews.csv` file is pulled
it needs to be removed from git

**Pull request closed without merging**